### ISSUE TITLE
Support for LPD8806 Led strips...

### ISSUE THREAD

**Issue by Snille** [EricHigdon/OctoPrint-RGB_status#36]
Hi! I'm sitting on a bunch of LPD8806-strips. Is there anyway to be able to control these somehow? Or maybe add support for them? :)

Thank you!

Best regards Erik

**Comment by Snille** [comment]
Oh, found this: https://github.com/Sh4d/LPD8806
Maybe it can be "included" some how to add support for LDP8806. :)

**Comment by EricHigdon** [comment]
Yes. I was considering something like that, but have to decide if I want to add more dependencies, or if there’s a better way.
